<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OpenSongImportDialog</class>
 <widget class="QDialog" name="OpenSongImportDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>747</width>
    <height>298</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OpenSong import</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../res/resources.qrc">
    <normaloff>:/icons/16/OpenSong_16px.png</normaloff>:/icons/16/OpenSong_16px.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="twGallery_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_2">
      <attribute name="icon">
       <iconset resource="../../res/resources.qrc">
        <normaloff>:/icons/16/Settings_16px.png</normaloff>:/icons/16/Settings_16px.png</iconset>
      </attribute>
      <attribute name="title">
       <string>Nastavení</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="7" column="3">
        <widget class="QComboBox" name="cmbConflictBehavior">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <property name="sizeAdjustPolicy">
          <enum>QComboBox::AdjustToContents</enum>
         </property>
         <item>
          <property name="text">
           <string>Nepřepisovat</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Přepsat</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="label_2">
         <property name="pixmap">
          <pixmap resource="../../res/resources.qrc">:/icons/16/Sort By Modified Date_16px.png</pixmap>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>nebo</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="8" column="3">
        <widget class="SongTagsLineEdit" name="lnTags"/>
       </item>
       <item row="3" column="1">
        <widget class="QRadioButton" name="optDirectoryTree">
         <property name="text">
          <string>Adresářová struktura:</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">bgImportStyle</string>
         </attribute>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label">
         <property name="pixmap">
          <pixmap resource="../../res/resources.qrc">:/icons/16/Timeline_16px.png</pixmap>
         </property>
        </widget>
       </item>
       <item row="3" column="3">
        <widget class="QPushButton" name="btnFolderStructure">
         <property name="icon">
          <iconset resource="../../res/resources.qrc">
           <normaloff>:/icons/16/Open_16px.png</normaloff>:/icons/16/Open_16px.png</iconset>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QPushButton" name="btnSelectFiles">
         <property name="icon">
          <iconset resource="../../res/resources.qrc">
           <normaloff>:/icons/16/Open_16px.png</normaloff>:/icons/16/Open_16px.png</iconset>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QRadioButton" name="optSingleSongs">
         <property name="text">
          <string>Jednotlivé písně:</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">bgImportStyle</string>
         </attribute>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Přidat štítky:</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Existující písně:</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1" colspan="3">
        <widget class="QCheckBox" name="cbAddToPlaylist">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Přidat importované písně do programu promítání</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>16</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_6">
         <property name="pixmap">
          <pixmap resource="../../res/resources.qrc">:/icons/16/Audio File_16px.png</pixmap>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="label_4">
         <property name="pixmap">
          <pixmap resource="../../res/resources.qrc">:/icons/16/Tags_16px.png</pixmap>
         </property>
        </widget>
       </item>
       <item row="7" column="2">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>16</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_10">
         <property name="pixmap">
          <pixmap resource="../../res/resources.qrc">:/icons/16/Folder_16px.png</pixmap>
         </property>
        </widget>
       </item>
       <item row="4" column="1" colspan="3">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_8">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="pixmap">
            <pixmap resource="../../res/resources.qrc">:/icons/16/Info_16px.png</pixmap>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_9">
           <property name="font">
            <font>
             <italic>true</italic>
            </font>
           </property>
           <property name="text">
            <string>V adresářové struktuře se prohledávají složky ve vybrané složce. Písním se přidá štítek dle názvu složky.</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>2</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnClose">
       <property name="text">
        <string>Storno</string>
       </property>
       <property name="icon">
        <iconset resource="../../res/resources.qrc">
         <normaloff>:/icons/16/Delete_16px.png</normaloff>:/icons/16/Delete_16px.png</iconset>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnImport">
       <property name="text">
        <string>Importovat</string>
       </property>
       <property name="icon">
        <iconset resource="../../res/resources.qrc">
         <normaloff>:/icons/16/Import_16px.png</normaloff>:/icons/16/Import_16px.png</iconset>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SongTagsLineEdit</class>
   <extends>QLineEdit</extends>
   <header>wgt/songtagslineedit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>btnSelectFiles</tabstop>
  <tabstop>cbAddToPlaylist</tabstop>
  <tabstop>lnTags</tabstop>
  <tabstop>btnClose</tabstop>
  <tabstop>btnImport</tabstop>
 </tabstops>
 <resources>
  <include location="../../res/resources.qrc"/>
 </resources>
 <connections/>
 <buttongroups>
  <buttongroup name="bgImportStyle"/>
 </buttongroups>
</ui>
